### ISSUE TITLE
Install script can abort on mkdir

### ISSUE THREAD

**Issue by solarizedalias** [kkoomen/vim-doge#113]
**Describe the bug**

Currently, `install.sh` (invoked by `doge#install()`) can fail if `$ROOT_DIR/bin` already exists (e.g. when users try to update the plugin). This happens because [`mkdir ./bin`](https://github.com/kkoomen/vim-doge/blob/3a397f26083ab18cdb8545662878244f5174e409/scripts/install.sh#L15) returns non-zero when the directory exists and [`set -e`](https://github.com/kkoomen/vim-doge/blob/3a397f26083ab18cdb8545662878244f5174e409/scripts/install.sh#L5) causes the entire script to abort, therefore results in empty `bin` directory (the binary, if exists, is removed every time the script runs).

**Settings**

**To Reproduce**
Run `call doge#install()` or `scripts/install.sh` more than one time.

**Expected behavior**
The script does not abort and the binary `vim-doge` is installed in `bin` properly.

**Comment by kkoomen** [comment]
I don't have which system you're running, but I don't have this problem on OSX, but I did add a check now. Shouldn't have this issue again in v3.3.1.

**Comment by solarizedalias** [comment]
Thank you, looks like it's solved.
Updated the plugin and confirmed `doge#install()` or running `install.sh` downloads the latest binary each time they run.

It's strange because I also use macOS (mojave), I tried in two versions of mkdir, `/bin/mkdir` and coreutils mkdir installed by homebrew by modifying `PATH`, which in this case behaved the same way. `mkdir: cannot create directory ‘./bin’: File exists`
